main.go

```diff
@@ -39,6 +38,18 @@ func rssHandler(rw http.ResponseWriter, r *http.Request) {
 	fmt.Fprintln(rw, result)
 }
 
+func fetchFeeds(master *feeds.Feed) {
+	var wg sync.WaitGroup
+	wg.Add(len(sourceFeeds))
```

Comment:
Though technically correct, its more common to call `wg.Add(1)` inside the for loop, just before the `go func(...)` call.